### DISCUSSION TITLE
Few ideas

### DISCUSSION THREAD

**Discussion by rxboy1983** [valinet/ExplorerPatcher#213] [Ideas]
Not requests, only some ideas:

- increase blur for file manager/context menu/taskbar (like in Windows Longhorn. Oryginal blur on Windows 11 is weak...)
- add blur on titlebars
- working TaskbarX in Windows 11 (I tested v. 1.7.2.0 and it not work on dev build)
- option to not show names for pined apps in Menu Start
- center captions in titlebar (I know that can it be done by custom theme, but works only for classic windows and file manager)
- option to remove icon in titlebar
- smaller new context menu (no results with editing svg files "C:\Windows\SystemApps\MicrosoftWindows.Client.CBS_cw5n1h2txyewy\FileExplorerExtensions\Assets\images\contrast-standard\...".I think that need change size of font)
- window "Settings of ExplorerPatcher" more integrated with Windows 11 (images geted from theme: Button>CHECKBOX/RADIOBUTTON/PUSHBUTTON)

**Comment by valinet** [comment]
Thanks. Although, a lot of these are more like separate applications in themselves, than additions for this project. I don't have a timeline for implementing any of them, plus I am super busy at the moment with some other stuff. These being said, regarding the ideas:
* Messing with DWM, so yeah...
* Again, messing with DWM...
* TaskbarX is not my software; that being said, I have already sent a PR to its author to fix compatibility with Windows 11: https://github.com/ChrisAnd1998/TaskbarX/pull/820, it's not my fault he/she is not updating the public releases with that; a custom compiled version I made was circulating around here at some point which fixed the issue: https://github.com/valinet/ExplorerPatcher/issues/13#issuecomment-932752027
* Haven't found yet a way to alter that UI tree the Start menu is using; don't really know a lot about the tech it uses to draw that, to be honest
* As a matter of fact, I already have a software for centering title bar text on my GitHub: https://github.com/valinet/WinCenterTitle; of course, works only for title bars drawn by DWM; since you keep wanting to mess with the DWM, I previously worked on enabling blur on title bars, but never got to finishing off the work: https://github.com/valinet/WinCenterTitle/tree/glass - maybe you are interested to pick up where i left off...
* Could somehow be done exploiting built-in functionality achieveable using `SetWindowThemeAttribute`
* Haven't researched that too much...
* Maybe in some distant future; the code at the moment is functional and non important imo; I don't exactly know how to load the images from the theme file, show them there, have to refactor the code etc; I don't have the time/energy for that, to be honest; if you want to do it and know how to do it, then please, fork the project, change the code, implement this and I will gladly accept a PR

Thank you
